.github/workflows/cli-test-npm.yml

```diff
@@ -0,0 +1,38 @@
+name: CLI Tests on npm installations
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to use for running test cases'
+        required: true
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16.x, 14.x, 10.x]
+    steps:
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+         node-version: ${{ matrix.node-version }}
+      -  name: Global - Testing installation of Twilio CLI (1)
+         run: |
+          npm install -g twilio-cli@${{ github.event.inputs.version }}
```

Comment:
Do we need to input the version number here? Npm should give us the latest version by default

Reply:
Yes, the idea is to test the version just released. The `version-draft` tag